molecule/provisioners/dockerprovisioner.py

```diff
@@ -113,7 +113,7 @@ def build_image(self):
             dockerfile = '''
             FROM {}:{}
             RUN bash -c 'if [ -x "$(command -v apt-get)" ]; then apt-get update && apt-get install -y python sudo; fi'
-            RUN bash -c 'if [ -x "$(command -v yum)" ]; then yum makecache fast && yum update -y && yum install -y python sudo; fi'  # NOQA
+            RUN bash -c 'if [ -x "$(command -v yum)" ]; then yum makecache fast && yum update -y && yum install -y python sudo; fi'
 
```

Comment:
Looks like my flake8 was old, the existing noqa in the code works for me now. This change was simply removing one that I had added. There's zero net change to that file.